extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -982,12 +984,49 @@ export class AmpDatePicker extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handle mousedown to prevent the mobile keyboard
+   * @param {!Event} e
+   * @private
+   */
+  addTouchReadonly_(e) {
```

Comment:
The naming is confusing here, then function name says touch, the comment says mouse but it seems to apply to both (as well as other sources of focus like keyboard).

Reply:
Changed the naming elsewhere to clarify